addon/globalPlugins/pcKbBrl.py

```diff
@@ -367,15 +411,111 @@ def makeSettings(self, settingsSizer):
 			self.cancelKeysEdit.SetValue(config.conf["pcKbBrl"]["cancelKeys"])
 		except KeyError:
 			pass
+		# Translators: label of a dialog.
+		dot1KeysLabel = _("Type the characters for &dot1 when writing with one hand.")
+		self.dot1KeysEdit = sHelper.addLabeledControl(dot1KeysLabel, wx.TextCtrl)
+		try:
+			self.dot1KeysEdit.SetValue(config.conf["pcKbBrl"]["dot1"])
+		except KeyError:
+			pass
```

Comment:
With the fix in the config spec, the try/catch statement should not be required anymore; suggestion for dot1 but should be done the same way for dots 2 to 8 and nullKey.

```suggestion
		self.dot1KeysEdit.SetValue(config.conf["pcKbBrl"]["dot1"])
```